internal/httpapi/server_test.go

```
@@ -2897,6 +2898,47 @@ func TestEventStreamUsesIncrementalReads(t *testing.T) {
+	listener, err := net.Listen("tcp", "127.0.0.1:0")
+	if err != nil {
+		t.Fatal(err)
+	}
+	server := NewServer(st)
+	httpServer := &http.Server{
+		Handler:     authenticatedMemoryHandler(server),
+		BaseContext: func(net.Listener) context.Context { return baseCtx },
+	}
+	serveDone := make(chan error, 1)
+	go func() { serveDone <- httpServer.Serve(listener) }()
+	response, err := http.Get("http://" + listener.Addr().String() + "/v1/tasks/shutdown-stream/events/stream") //nolint:gosec
```

golangci-lint (2.13.2)

[error] response body must be closed

(bodyclose)